plugin/security/src/main/java/org/opencds/cqf/ruler/plugin/security/OAuthConfig.java

```diff
@@ -0,0 +1,24 @@
+package org.opencds.cqf.ruler.plugin.security;
+
+import ca.uhn.fhir.jpa.starter.annotations.OnR4Condition;
+
+import org.hl7.fhir.r4.model.CapabilityStatement;
+import org.opencds.cqf.ruler.api.MetadataExtender;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Conditional;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class OAuthConfig {
+    
+    @Bean
+	 @ConfigurationProperties(prefix = "oauth")
```

Comment:
Let's called this class "SecurityConfig" since it'll be used for things other than the OAuth meta-data in the future. Let's also use the prefix "hapi.fhir.ouath".